test/react-native/features/fixtures/app/scenarios/UnhandledNativeErrorScenario.js

```diff
@@ -1,8 +1,8 @@
-import Scenario from "./Scenario";
-import { NativeModules } from "react-native";
+import Scenario from './Scenario'
+import { NativeModules } from 'react-native'
 
-export default class UnhandledNativeErrorScenario extends Scenario {
-    run() {
-        NativeModules.BugsnagTestInterface.runScenario("UnhandledExceptionScenario", () => {})
-    }
-}
\ No newline at end of file
+export class UnhandledNativeErrorScenario extends Scenario {
+  run() {
+    NativeModules.BugsnagTestInterface.runScenario('UnhandledNativeErrorScenario', () => {})
```

Comment:
Just a thought for a future changes, we could grab the scenario name from the class name here to avoid typos or copy/paste errors creeping in.